packages/@aws-cdk/aws-neptune/test/integ.cluster.ts

```diff
@@ -43,6 +44,14 @@ const cluster = new DatabaseCluster(stack, 'Database', {
 
 cluster.connections.allowDefaultPortFromAnyIpv4('Open to the world');
 
+const metric = cluster.metric('SparqlErrors', { statistic: cloudwatch.Statistic.SUM });
```

Comment:
It looks like `SparqlErrors` has been deprecated, can you update this test to use another metric?

https://docs.aws.amazon.com/neptune/latest/userguide/cw-metrics.html#cw-metrics-deprecated